### PR TITLE
Added Managed Instance extension

### PR DESCRIPTION
**Pull request by JocaPC**

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -1534,6 +1534,63 @@
...
+					},
+					"versions": [
+						{
+							"version": "0.1.1",
+							"lastUpdated": "4/30/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
```
Comment (kburtram) [inline extensionsGallery.json]: We currently don't support in-product installation of non-Microsoft published extensions due to policy constraints.  We're looking at various marketplace options to support this in the future, but currently external extensions need to be published with `"assetType": "Microsoft.SQLOps.DownloadPage"` and a link to a download page.

Comment (JocaPC) [reply extensionsGallery.json]: Updated, thanks.

**Comment by kburtram** [comment]
@alanrenmsft let's publish this as a Microsoft extension.

File: extensionsGallery.json [hunk 2]
```diff
@@ -1534,6 +1534,63 @@
...
+					},
+					"versions": [
+						{
+							"version": "0.1.1",
+							"lastUpdated": "4/30/2019",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
```
Comment (abist) [inline extensionsGallery.json]: We should have `AzureDataStudio` instead of `SQLOps` or `VisualStudio.Code` in assetTypes

**Comment by yualan** [comment]
Will @JocaPC need to make these changes?

**Comment by alanrenmsft** [comment]
@yualan no, I have published these changes yesterday.

**Pull request closed without merging**